includes/ucf-degree-search-custom-filters.php

```diff
@@ -92,5 +92,24 @@ public static function valid_taxonomy( $taxonomy ) {
 
 			return taxonomy_exists( $taxonomy );
 		}
+
+		/**
+		 * Custom where filter for filtering by career_paths
+		 */
+		public static function filter_by_career_paths( $prepared_args, $request) {
+			if ( isset( $request['career_paths'] ) ) {
```

Comment:
Could these if-statements be combined?

Reply:
No because there's a specific else statement that's supposed to be below that I never put in.